Content.Shared/SimpleStation14/Species/Shadowkin/Systems/ShadowkinSystem.Blackeye.cs

```diff
@@ -0,0 +1,26 @@
+using Content.Shared.SimpleStation14.Species.Shadowkin.Components;
+using Content.Shared.SimpleStation14.Species.Shadowkin.Events;
+
+namespace Content.Shared.SimpleStation14.Species.Shadowkin.Systems
+{
+    public sealed class ShadowkinBlackeyeSystem : EntitySystem
+    {
+        [Dependency] private readonly IEntityManager _entityManager = default!;
+
+        public override void Initialize()
+        {
+            base.Initialize();
+
+            SubscribeAllEvent<ShadowkinBlackeyeEvent>(OnBlackeye);
+        }
+
+        private void OnBlackeye(ShadowkinBlackeyeEvent ev)
+        {
+            // Remove powers
+            _entityManager.RemoveComponent<ShadowkinDarkSwapPowerComponent>(ev.Uid);
+            _entityManager.RemoveComponent<ShadowkinDarkSwappedComponent>(ev.Uid);
+            _entityManager.RemoveComponent<ShadowkinRestPowerComponent>(ev.Uid);
+            _entityManager.RemoveComponent<ShadowkinTeleportPowerComponent>(ev.Uid);
+        }
```

Comment:
Consider looking into the status effect system or whatever it's called. Could be interesting here?